src/types.rs

```diff
@@ -148,11 +148,19 @@ pub enum JoinType {
     RightJoin,
 }
 
+/// Nulls order
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Nulls {
```

Comment:
I think we don't need to pub this ? We can keep this as `pub(crate)`?
If not, we need to think of a better name than `Nulls`.
I am thinking may be `NullOrdering` or something

Reply:
1. If this keep as pub(crate), how do you think users should use these methods order_by_xxxs_with_nulls?
  I think it need pass bool as nulls order argument ?
2. I also think `NullOrdering` is better than `Nulls`